packages/contentstack-migrate-rte/package.json

```diff
@@ -6,7 +6,7 @@
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "^1.0.1",
-    "@contentstack/cli-utilities": "^1.0.2",
+    "@contentstack/cli-utilities": "^1.0.3",
```

Comment:
We need to update this package version and all location where this package is used

Reply:
done